src/assignment/views/AssignmentResponses.tsx

```diff
@@ -243,8 +258,18 @@ const AssignmentResponses: FunctionComponent<AssignmentResponsesProps> = ({
 				query.filter || ''
 			);
 
+			// Determine each response's fragments to evaluate their published-status
+			const fragments = response.assignmentResponses
+				.flatMap((response) =>
+					((response.pupil_collection_blocks as PupilCollectionFragment[]) || []).flatMap(
+						(block) => block.fragment_id
+					)
+				)
+				.filter((value, index, self) => self.indexOf(value) === index);
```

Comment:
what does this do?

Reply:
Prevents duplicate string values in the payload sent when querying for items.

Reply:
mss is lodash uniq of uniqBy dan een duidelijkere oplossing:
https://lodash.com/docs/4.17.15#uniqBy